text/0004-enriched-indicator-value.md

```diff
@@ -0,0 +1,63 @@
+# Enriched Service Level Indicators
+
+Enrich Service Level Indicator (SLI) with a set of values and a deep link to the data source.
+
+## Motivation
+
+In the current approach of implementing an SLI-provider*, the provider has to send out a Keptn CloudEvent of type [sh.keptn.internal.event.get-sli.done](https://github.com/keptn/spec/blob/master/cloudevents.md#get-sli-done) after querying the data from its data source. In the payload of this event, an array of indicators and their value is provided, e.g.:
+
+```yaml
+    "indicatorValues": [
+      {
+        "metric":"request_latency_p95",
+        "value": 1.1620000000000001,
+        "success": true
+      },
+      {
+        "metric":"error_rate",
+        "value": 0,
+        "success": true
+      }
+    ],
+```
+
+The limitation is now that describing an SLI by one metric is too restricted. Most likely, more data points for an indicator are available that would provide an even better understanding of the indicator and support the visualization of the SLI. Additionally, a link for drilling into the tool (that is queried by the SLI provider) is useful to get a look at the actual (real) data set.
+
+(* An SLI-provider is a vendor-specific service that queries data from a data source (e.g, from a monitoring or testing solution) using an SLI configuration.)
+
+## Explanation
+
+Instead of limiting an indicator to a single value, this KEP proposes to enrich the event payload of [sh.keptn.internal.event.get-sli.done](https://github.com/keptn/spec/blob/master/cloudevents.md#get-sli-done) with a set of values and a deep link for each indicator, as shown below:
+
+```yaml
+    "indicatorValues": [
+      {
+        "metric":"request_latency_p95",
+        "values": [1.162, 1.651, 1.162, 1.653, 1.192, 1.191],
```

Comment:
Where did `value` go? I believe we should have both, `value` as a required field, and `values` should be optional.

In addition, just `values` would be the `y` axis, I think we would also need an `x` axis for those values, or at least define  that they are evenly spread between `start` and `end` datetimes of the get-sli event.